pages/delegates/[network]/[address].tsx

```diff
@@ -83,11 +84,13 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 };
 
 export const getStaticPaths: GetStaticPaths = async () => {
-  const delegates = await fetchDelegates();
-  const paths = delegates.map(d => `/delegates/${d.voteDelegateAddress}`);
+  const delegatesMainnet = await fetchDelegates(SupportedNetworks.MAINNET);
+  const delegatesKovan = await fetchDelegates(SupportedNetworks.KOVAN);
```

Comment:
Kind of the same comment I made above but this feels like a waste of a call, I don't think many people (besides us) will be using kovan very often in production.

Reply:
We must fetch both if we want to use getStaticPaths and getStaticProps. If we don't want paths and props we can fetch only on demand on the client side.